docs/develop.md

```diff
@@ -38,7 +38,7 @@ $ yarn start
 
 ### Add E2E test cases
 
-Please refer to [E2E Documentation](../frontend/src/e2e/README.md).
+Please refer to [E2E Documentation](../web/src/e2e/README.md).
```

Comment:
We would better rename all `frontend` to `web` , such as L22

Reply:
Let's rename them